### ISSUE TITLE
Disabling blocks not working?

### ISSUE THREAD

**Issue by parijke** [drupalprojects/kalatheme#103]
Hi all,

I have a fresh installation of drupal with the latest dev. With that fresh install I created a subtheme using drush "My awsome theme". Also enable panopoly_theme.

So far so good.

I created a panels page using page manager, checking "Disable Drupal blocks/regions" and created a two column page with the Burr layout.

The page looks nicely except that the blocks like navigation are still visible (and outside the container)

![image](https://cloud.githubusercontent.com/assets/7671998/3454839/02f54a04-01d7-11e4-8dd9-4c26e3dc4253.png)

Is this a bug?

**Comment by pirog** [comment]
Latest dev of what branch?

**Comment by parijke** [comment]
Hi Mike,

Kalatheme 7.x-4.x-dev

**Comment by pirog** [comment]
@parijke 4.x dev is still a work in progress but @bassettsj may have more insight here

**Comment by bassettsj** [comment]
@parijke did you disable the blocks module itself?

**Comment by parijke** [comment]
Hi Steven,

No I didn't. I bet it will work then bu I need blocks sometimes

**Comment by pirog** [comment]
Not 100% sure but i _think_ you can have blocks on and just move the blocks
out of the content region in the blocks UI?

On Wed, Jul 2, 2014 at 10:08 AM, parijke notifications@github.com wrote:

> Hi Steven,
> 
> No I didn't. I bet it will work then bu I need blocks sometimes
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/drupalprojects/kalatheme/issues/103#issuecomment-47806275
> .

## 

Cheers,

Mike Pirog
Kalamuna
www.kalamuna.com

**Comment by bassettsj** [comment]
I would really recommend _not_ using blocks with a panels based site, you have two competing paradigms here.

If you have to have blocks, remove the blocks on any panels page from the blocks management page.

**Comment by pirog** [comment]
+1

On Wed, Jul 2, 2014 at 10:11 AM, Steven Bassett notifications@github.com
wrote:

> I would really recommend _not_ using blocks with a panels based site, you
> have two competing paradigms here.
> 
> If you have to have blocks, remove the blocks on any panels page from the
> blocks management page.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/drupalprojects/kalatheme/issues/103#issuecomment-47806527
> .

## 

Cheers,

Mike Pirog
Kalamuna
www.kalamuna.com

**Comment by parijke** [comment]
Hi Steven,

Yes I mostly aggree on being reluctant with blocks. But sometimes it is needed.

I checked the blocks configuration page (for the My Awesome Theme Kalatheme Subtheme) and most of the enabled block were in the content region.

I removed them from the content region to get rid of them, but I think it should be working with "Disable blocks/regions" from the panels page configuration.

I will look into it how other themes have implemented this.
